app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/Router.scala

```diff
@@ -181,7 +186,7 @@ trait JourneyTypeRoutes extends Product with Serializable {
           case false => claimRoutes.SelectBasisForClaimController.selectBasisForClaim(journeyBindable)
         }
       case NoClaimantDetailsAnswer  =>
-        claimRoutes.CheckContactDetailsMrnController.show(journeyBindable)
+        claimRoutes.CheckContactDetailsMrnController.addShow(journeyBindable)
```

Comment:
to be consistent with the other naming convention used elsewhere, `addShow` should be something like `showAddContactDetails` and the submit method `showAddContactDetailsSubmit` or something else that follows that pattern.